mobileapp/www/js/index.js

```diff
@@ -6,14 +6,25 @@
 
 document.addEventListener("deviceready", onDeviceReady, false);
 
+// Global variables on the fly
+// Intended for simple boolean values, NOT for passing details!
+// Usage: if(thing_to_set) {GLOBALS.foo = [true || false]}.
+var GLOBALS = {};
```

Comment:
you should not use the name "GLOBALS" - rather a name that better describes what these values are. Also, never create a global variable if it can be helped. This object should sit inside the "app" object